### PR TITLE
Adds support for scaling image sizes when added/edited

### PR DESCRIPTION
**Pull request by kharbutli-meta**
## Description

Adds support, in both the backend and frontend, for scaling image sizes when added/edited.

## Testing

* Unit tests: Updated.
* Ran locally.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/src/containers/AddImage.tsx [hunk 1]
```diff
@@ -217,6 +225,74 @@ function AddImage() {
...
+                          label: t("AddImageScreen.25"),
+                        },
+                      ]}
+                    />
+                  </div>
+                  <div className="grid-col-3 margin-top-10">
+                    <div style={{ position: "absolute", bottom: "0px" }}>
+                      {scalePct === "100%" && (
+                        <img
+                          src={`${process.env.PUBLIC_URL}/images/image-scale-100.svg`}
```
Comment (mabreuortega) [inline frontend/src/containers/AddImage.tsx]: recomendation: refactor URL resolution to a method and create only one image component

```
<img
    src={getScaleUrl(scalePct)}
    width="100%"
    height="auto"
  />
```

File: frontend/src/components/ImageWidget.tsx [hunk 2]
```diff
@@ -7,10 +7,11 @@ type Props = {
   file: File | undefined;
   summaryBelow: boolean;
   altText: string;
+  scalePct: string;
```
Comment (mabreuortega) [inline frontend/src/components/ImageWidget.tsx:10]: recommendation: consider restricting type to allow only possible values

```
scalePct: "auto" | "25%" | "50%" | "75%" | "100%"
```

or use an enum type

```
export enum ScalePct {
   Auto: "auto"
...
}
```

File: frontend/src/locales/en/translation.json [hunk 3]
```diff
@@ -896,7 +896,14 @@
     "ToggleSummary": "Show description below image",
     "Back": "Back",
     "DisabledToolTip": "Upload a file to continue",
-    "Cancel": "Cancel"
+    "Cancel": "Cancel",
+    "ImageSize": "Image size (optional)",
+    "ImageSizeHint": "How your image will display in the dashboard.",
+    "AsUploaded": "As uploaded (do not scale)",
+    "100": "100%",
```
Comment (mabreuortega) [inline frontend/src/locales/en/translation.json]: suggestion: use interpolation

```
"percentage": "{percent}%"
```

and use

```
t("percentage", { percent: scalePct })
```

Comment (mabreuortega) [reply frontend/src/locales/en/translation.json]: this percentage can be global and you can reuse across the app. x% have no changing context

**Review by mabreuortega — APPROVED** [review]
LGTM, multiple suggestion made.

File: frontend/src/containers/AddImage.tsx (hunk 1 above)
Comment (kharbutli-meta) [reply frontend/src/containers/AddImage.tsx]: Good recommendation. I changed it to use an inline function.

File: frontend/src/components/ImageWidget.tsx (hunk 2 above)
Comment (kharbutli-meta) [reply frontend/src/components/ImageWidget.tsx:10]: I prefer to keep it as is for two reasons:
1. Makes it easier to add values in the future especially if we decide to allow a user to enter any scale value.
2. Makes other parts of the code simpler as we can insert the scale string value immediately as is.

File: frontend/src/locales/en/translation.json (hunk 3 above)
Comment (kharbutli-meta) [reply frontend/src/locales/en/translation.json]: Done.

**Pull request merged**